apps/api/src/app/integrations/dtos/update-integration.dto.ts

```diff
@@ -37,14 +37,6 @@ export class UpdateIntegrationRequestDto implements IUpdateIntegrationBodyDto {
   @ValidateNested()
   credentials?: CredentialsDto;
 
-  @ApiPropertyOptional({
-    type: Boolean,
-    description: 'If true, the Novu branding will be removed from the Inbox component',
-  })
-  @IsOptional()
-  @IsBoolean()
-  removeNovuBranding?: boolean;
-
```

Comment:
Breaking change in theory but in pratice noone was updating branding via integrations API so it shouldn't matter